lib/src/lib.rs

```diff
@@ -8,7 +8,11 @@
 //
 
 #[macro_use] extern crate log;
+#[macro_use] extern crate error_chain;
 extern crate git2;
 
+pub mod error;
 pub mod repository;
 
+mod first_parent_iter;
```

Comment:
I'm rather sure we need to make this `pub` or export the iterator in another way... But I guess we can do so later.